karrot/places/receivers.py

```diff
@@ -47,6 +47,8 @@ def subscription_removed(sender, instance, **kwargs):
 @receiver(pre_delete, sender=GroupMembership)
 def group_member_removed(sender, instance, **kwargs):
     membership = instance
+    membership.group
```

Comment:
I guess this avoids lazy access in the loop?

Reply:
Ah, that was unintentional. I think maybe one of the `pre-commit` hooks removed itself assignment to a local variable, but left it there (incase it had side effects?). Removed in latest commit.